skrifa/src/outline/glyf/hint/engine/mod.rs

```diff
@@ -47,6 +47,45 @@ pub struct Engine<'a> {
 }
 
 impl<'a> Engine<'a> {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
I mean, clippy has my sympathies here...

Reply:
and mine... I actually wrote the clippy allow before the arguments because I knew what was coming. I'll likely just make all the fields in this thing public in the future anyway. I have aspirations to expose the guts of the hinting code to allow for debugging in which case users will likely want to poke at all the internal state